### PR TITLE
[spdlog] Update to 1.9.2

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Updates spdlog to 1.9.2

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  as before, Yes

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

File: ports/spdlog/vcpkg.json [hunk 1]
```diff
@@ -1,7 +1,6 @@
 {
   "name": "spdlog",
-  "version-semver": "1.9.0",
-  "port-version": 1,
+  "version-semver": "1.9.2",
```
Comment (JonLiu1993) [inline ports/spdlog/vcpkg.json:3]: @chausner ，Could you confirm whether this version follows the definition of version-semver, if not, please change to version

Comment (chausner) [reply ports/spdlog/vcpkg.json:3]: The developer confirms it is following semantic versioning: https://github.com/gabime/spdlog/issues/86#issuecomment-925738767

Comment (JonLiu1993) [reply ports/spdlog/vcpkg.json:3]: Thanks for your clarification

**Review by JonLiu1993 — APPROVED** [review]

**Comment by BillyONeal** [comment]
Thanks for the update!

**Pull request merged**